package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "koishi-plugin-forward",
+  "name": "@constasj/koishi-plugin-forward",
```

Comment:
You should NOT change the name of the plugin.

Reply:
I made a mistake.
Maybe I should checkout a new branch.